sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,17 +1,23 @@
 # Release History
 
-## 3.1.0-beta.4 (Unreleased)
+## 3.1.0 (Unreleased)
+### Features Added
+- Added `clientOptions()` and `getDefaultLogOptions()` methods to the `FormRecognizerClientBuilder` and `FormTrainingClientBuilder`.
+- We are able to support multiple service API versions now: `V2_0` and `V2_1`.
+- Add more static values to `FormRecognizerLanguage` expandable string class.
+
 ### Breaking Changes
-- Renamed `ReadingOrder` model to `FormReadingOrder`.
+- The client defaults to the latest supported service version, which currently is 2.1.
```

Comment:
should the removal of FieldValueType "gender" and "country" and addition of "countryRegion" be added here?